eth/accountmanager.go

```diff
@@ -129,19 +130,20 @@ func (am *accountManager) CreateTransactOpts(gasLimit uint64, gasPrice *big.Int)
 				return nil, errors.New("not authorized to sign this account")
 			}
 
-			return am.SignTx(signer, tx)
+			return am.SignTx(tx)
 		},
 	}, nil
 }
 
 // Sign a transaction. Account must be unlocked
-func (am *accountManager) SignTx(signer types.Signer, tx *types.Transaction) (*types.Transaction, error) {
-	signature, err := am.keyStore.SignHash(am.account, signer.Hash(tx).Bytes())
+func (am *accountManager) SignTx(tx *types.Transaction) (*types.Transaction, error) {
+	// Ignore signer argument and use EIP155Signer instance instead
```

Comment:
Don't think we need this comment for the same reason as my comment earlier in this review about the caller configuring the `Signer`.

Reply:
fixed in aee8f653